velox/docs/functions/json.rst

```diff
@@ -133,6 +133,13 @@ JSON Functions
         SELECT json_size('{"x": [1, 2, 3]}', '$.x'); -- 3
         SELECT json_size('{"x": {"a": 1, "b": 2}}', '$.x.a'); -- 0
 
+.. function:: json_format(json) -> varchar
```

Comment:
Functions in the documentation should appear in alphabetical order. Please, move this function accordingly.